jenkins/infra/terraform/locals.tf

```diff
@@ -0,0 +1,15 @@
+locals {
+  ip_subnet_mapping = [{
```

Comment:
Where do these names come from? Are these resources also managed in Terraform? If so, can we get them from the Terraform state instead of hardcoding them?

Reply:
They were created through ClickOps. If you want I can set them up here and import them.